plat/imx/common/imx_sip_handler.c

```diff
@@ -154,6 +157,11 @@ int imx_src_handler(uint32_t smc_fid,
 		    void *handle)
 {
 	uint32_t val;
+#if defined(PLAT_imx8mn) || defined(PLAT_imx8mp)
+	uint64_t timeout;
+	int ret1 = 0, ret2 = 0;
```

Comment:
There is no need to use 2 return values here, rather than you can update the return parameter via `SMC_SET_GP(handle, CTX_GPREG_X[1,2], ret)` at the end of each `while wait` loop